### PR TITLE
autojump 22.2.4

### PR DESCRIPTION
**Pull request by youtux**
The install script has been fixed and we can use it.

### CODE REVIEW

File: Library/Formula/autojump.rb [hunk 1]
```diff
@@ -1,42 +1,40 @@
...
 
-    bin.write_exec_script libexec+"autojump"
+    libexec.install bin
+    bin.write_exec_script libexec/"bin/autojump"
   end
 
   def caveats; <<-EOS.undent
     Add the following line to your ~/.bash_profile or ~/.zshrc file (and remember
     to source the file to update your current session):
-      [[ -s $(brew --prefix)/etc/autojump.sh ]] && . $(brew --prefix)/etc/autojump.sh
```
Comment (mikemcquaid) [inline Library/Formula/autojump.rb]: `$(` is actually preferably over backticks.

File: Library/Formula/autojump.rb [hunk 2]
```diff
@@ -1,42 +1,40 @@
...
+      [[ -s `brew --prefix`/etc/profile.d/autojump.sh ]] && . `brew --prefix`/etc/profile.d/autojump.sh
 
-    Add the following line to your ~/.config/fish/config.fish:
-      . #{etc}/autojump.fish
+    If you use the Fish shell then add the following line to your ~/.config/fish/config.fish:
+      if test -f #{HOMEBREW_PREFIX}/share/autojump/autojump.fish; . #{HOMEBREW_PREFIX}/share/autojump/autojump.fish; end
```
Comment (mikemcquaid) [inline Library/Formula/autojump.rb]: `[ -f #{HOMEBREW_PREFIX}/share/autojump/autojump.fish ] && . #{HOMEBREW_PREFIX}/share/autojump/autojump.fish`

File: Library/Formula/autojump.rb (hunk 1 above)
Comment (xu-cheng) [inline Library/Formula/autojump.rb]: I don't like such change. As in fact it breaks backwards compatibility. Users upgrade their `autojump`, then suddenly it's not working until they realize they should change their shell configuration.

Comment (youtux) [inline Library/Formula/autojump.rb]: Line 13 should avoid it. I've tested it.

Comment (xu-cheng) [inline Library/Formula/autojump.rb]: :+1:

**Pull request closed without merging**